juriscraper/opinions/united_states/federal_district/dcd.py

```diff
@@ -4,142 +4,102 @@
-                # list sufficiently
-                count = len(row.xpath(urlpath))
-                dates.extend([date_object] * count)
-        return dates
 
-    def _get_precedential_statuses(self):
-        return ["Published"] * len(self.case_names)
+    def get_values_from_url(self, url: str) -> Tuple[str, str]:
+        """Get docket document number and nature_of_suit values from URL
 
-    def _get_docket_numbers(self):
-        docket_numbers = []
-        rowpath = "//table[2]//tr[position()>0]"
-        dktpath = "./td[2]//text()[following-sibling::br]"
-        urlpath = "./td[3]/a/@href"
-        for row in self.html.xpath(rowpath):
-            docket_number = row.xpath(dktpath)
-            # Determine the number of urls in each row and pad the docket
-            # numbers list sufficiently
-            count = len(row.xpath(urlpath))
-            docket_numbers.extend(docket_number * count)
-        return docket_numbers
+        :param url:
+        :return:  docket document number and nature_of_suit
+        """
+        # In 2012 (and perhaps elsewhere) they have a few weird urls.
+        match = self.docket_document_number_regex.search(url)
+        if match:
+            doc_number = match.group(6)
+        else:
+            doc_number = url
 
-    def _get_docket_document_numbers(self):
-        document_numbers = []
-        regex = re.compile(r"(\?)(\d+)([a-z]+)(\d+)(-)(.*)")
-        for url in self.html.xpath(
-            "//table[2]//tr[position()>0]/td[3]/a/@href"
-        ):
-            # Because we are acting directly on the entire url list, no padding
-            # of the docket number field is required.
-            doc_no = regex.search(url)
-            # In 2012 (and perhaps elsewhere) they have a few weird urls.
-            if re.search(regex, url) is not None:
-                document_numbers.append(doc_no.group(6))
+        nature_of_suit_match = re.search(self.nature_of_suit_regex, url)
+        # In 2012 (and perhaps elsewhere) they have a few weird urls.
+        if not nature_of_suit_match:
+            nature_of_suit = "Unknown"
+        else:
+            nature_code = nature_of_suit_match.group(3)
+            if nature_code == "cv":
+                nature_of_suit = "Civil"
+            elif nature_code == "cr":
+                nature_of_suit = "Criminal"
+            # This is a tough call. Magistrate Cases are typically also
+            # Criminal or Civil cases, and their docket_number field will
+            # reflect this, but they do classify these separately under
+            # these 'mj' and 'mc' codes and the first page of these
+            #  documents will often refer to them as 'Magistrate Case
+            # ####-####' so, we will too.
+            elif nature_code == "mj" or "mc":
+                nature_of_suit = "Magistrate Case"
             else:
-                document_numbers.append(url)
-        return document_numbers
+                nature_of_suit = "Unknown"
```

Comment:
yeah - I think thats probably incorrect - and that we should do nature of suits if we can identify either the nature of suit code - or the name value for one of those nature of suits.  otherwise I feel like we are muddying the database.

Reply:
So, should I remove it entirely from this scraper? In `dcd` the datapoint "Civil", "Criminal", "Magistrate Case" is too general to get a code from the document

Also, should we track this in bigger issue? We would need to correct more scrapers, and the data already in the DB.

Reply:
yes I think so.  Its misleading and currently not a field we have